From a01709456f532936ea5fdb0b834f412fe91b1d77 Mon Sep 17 00:00:00 2001
From: CoryXie <cory.xie@gmail.com>
Date: Sun, 6 Oct 2013 15:57:27 +0800
Subject: [PATCH] Enable system for input events

---
 ueventd.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ueventd.rc b/ueventd.rc
index 2cf0265..eda4ce2 100644
--- a/ueventd.rc
+++ b/ueventd.rc
@@ -33,7 +33,7 @@
 /dev/tty0                 0660   root       system
 /dev/graphics/*           0660   root       graphics
 /dev/msm_hw3dm            0660   system     graphics
-/dev/input/*              0660   root       input
+/dev/input/*              0660   system     input
 /dev/eac                  0660   root       audio
 /dev/cam                  0660   root       camera
 /dev/pmem                 0660   system     graphics
-- 
1.8.1.2

